package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.3",
+  "version": "2.0.0-0",
```

Comment:
Why isn't the version number just `2.0.0`? Why the extra `-0`?

Reply:
Because it is the first release candidate to the 2.0.0 (pre-release).
https://github.com/layerx-labs/taikai-locales/releases/tag/v2.0.0-0